deep-learning/_posts/2018-01-17-SBNet.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: review
+title:  "SB-Net: Sparse Block Network for Fast Inference"
+tags:   "Deep Learning"
+author: Faezeh Amjadi
+pdf:    https://arxiv.org/abs/1801.02108   
+cite:
+  authors: "Mengye Ren, Andrei Pokrovsky, Bin Yang, Raquel Urtasun"
+  title:   "SB-Net: Sparse Block Network for Fast Inference"
+  venue:   "--"
+---
+
+### Description
+
+The authors proposed a novel non-sparse convolution algorithm. They use a mask that is known and can apply to input images. For example, in a autonomous driving, only the areas on the roads are interested and be accessible from the offline road map mode. This mask is a binary {0-1} to represent active/inactive  regionsof images for CNN, and all CNN operations, will be placed on the active part of images. The binary mask has been converted to a list of indices, where each index belongs to the location of the corresponding pixel (Fig:2 ).  
```

Comment:
non-sparse -> sparse

Reply:
interested -> interesting

Reply:
road map mode -> road map

Reply:
The convolution gets a non sparse parts of an image, because a mask takes important parts of an input image and other parts is deleted.

we have two mode: offline / online 
Also, I have to mention that the blocks are sparse because they work with index.